src/task/DoAfterTasks.java

```diff
@@ -0,0 +1,22 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by Fernflower decompiler)
+//
+
+package task;
+
+import java.io.Serializable;
+
+public class DoAfterTasks extends task.Task implements Serializable {
+    protected String after;
+
+    public DoAfterTasks(String description, String after) {
+        super(description);
+        this.after = after;
+    }
+
+    public String GiveTask() {
+        String var10000 = super.GiveTask();
+        return "[A]" + var10000 + "(after: " + after + ")";
+    }
+}
```

Comment:
Remember to add some documentation to the things you add so that its easier for the rest to understand :) otherwise goodjob!